.github/workflows/gochecks.yml

```
@@ -40,3 +52,4 @@ jobs:
             echo "No Makefile test target, running tests with race detection as default behavior"
             go test -race ./...
           fi
+        shell: bash # to use bash on Windows
```

No if here. So bash is used everywhere

So it's more a


```suggestion
        shell: bash # force bash on Windows too, git-bash is installed a few steps before
```

Or something like this

---

bash is the default afaik for the others, so specifying it is really to say we can't run this code with powershell

---

I assumed it, I was raising the point a comment might help, but maybe you are the only maintainer of fortio/workflows :grin: